Framework/MDAlgorithms/src/IntegratePeaksMD2.cpp

```diff
@@ -477,11 +490,11 @@ template <typename MDE, size_t nd> void IntegratePeaksMD2::integrate(typename MD
 
     // Do not integrate if sphere is off edge of detector
 
-    double edge = detectorQ(p.getQLabFrame(), std::max(BackgroundOuterRadius[0], PeakRadius[0]));
-    if (edge < std::max(BackgroundOuterRadius[0], PeakRadius[0])) {
+    double edgeDist = calculateDistanceToEdge(p.getQLabFrame(), std::max(BackgroundOuterRadius[0], PeakRadius[0]));
```

Comment:
I seem to have got v enthusiastic about my const qualifiers this evening! I think edgeDist should be declared const double. Again - predates your changes